callout-code/src/main/java/execution.proto

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google LLC
+//
+// <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+// except in compliance with the License. You may obtain a copy of the License at
+//
+// <p>https://www.apache.org/licenses/LICENSE-2.0
+//
+// <p>Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package apigee;
+
+import "google/protobuf/any.proto";
+
+option java_package = "com.google.apigee";
+
+// Execution holds both the MessageContext and ExecutionContext objects
+// used for java callouts and policy execution.
+message Execution {
+  MessageContext messageContext = 1;
+  ExecutionContext executionContext = 2;
+}
+
+// MessageContext represents the processing of a runtime request, including
+// the request and response/error phases. For a request, the MessageContext
+// is shared across the execution of policies and Java Callouts.
+message MessageContext {
+  Message request_message = 1;
```

Comment:
Did we want to add proxy/target request/response messages here?

Reply:
Yes.  I forgot about it.  Thanks for pointing it out.

Reply:
yeah whoops, i forgot too. added.